package.json

```diff
@@ -38,13 +38,18 @@
     }
   },
   "dependencies": {
-    "revery": "revery-ui/revery#431e515a00ee46f7bc614a10c2a219aae13179e0",
-    "@opam/dune": "2.6.0",
+    "revery": "*",
+    "@opam/dune": "^2.6.0",
     "@revery/timber": "*",
     "esy-macdylibbundler": "*"
   },
+  "resolutions": {
+    "revery": "revery-ui/revery#d737fcd",
+    "esy-skia": "revery-ui/esy-skia#6106223",
+    "@esy-ocaml/reason": "EduardoRFS/reason:reason.json#35aa4df3de0daa60bdc1133dcf97855decac48f7"
+  },
   "devDependencies": {
-    "ocaml": "~4.9.0",
```

Comment:
Not sure if we should add "ocaml": "~4.12", to `dependencies` as well 🤔 

I see that's [done in oni](https://github.com/onivim/oni2/blob/1c951d4f92633dd9971706d8848fa69b40d4ba8e/package.json#L408)